Resolve #6 by adding settings via presets

JFHoelscher opened the pull request:
Add a list of settings that are not available via quick develop, for now containing _texture_ and _dehaze_. Upon editing a photo (and resetting the values at the end), check for each setting whether it is in this list. If so, adjust it by creating and applying a develop preset.

JFHoelscher committed:
Make dehaze and texture available

Adresses issue #6: Additional settings can now be adjusted. Implemented by creating and applying develop presets. For now, only added "dehaze" and "texture" to the table of available settings. In principle, this should work for all settings that are listed in the GUI when applying a develop preset.

JFHoelscher committed:
Update README.md

JFHoelscher commented:
I want to perform one more check whether the new functions work as intended, so better wait with the merge.

dekuenstle commented:
> I want to perform one more check whether the new functions work as intended, so better wait with the merge.

just get back to me when it's ready

JFHoelscher committed:
Fix: Actually apply changes in new settings

Changes via presets (i.e. texture and dehaze) were not actually applied due to faulty creation of the table of settings used to create the preset.

JFHoelscher commented:
There actually was a bug in the last version, I fixed it in the recent commit. Now ready for merging.

JFHoelscher committed:
Fix: correctly set new values using quick develop, solves wichmann-lab#7

For intended values different from 0, settings were not set to the intended value, but the intended value was added to the old value. This is fixed by resetting the value to 0 before the adjustment.

JFHoelscher commented:
Added a fix that solves #8.
